### PR TITLE
SAK-41782 - Accept all possible proper mimetypes for XML

### PR DESCRIPTION
**Pull request by csev**
This was originally reported in tsugi-util  https://github.com/tsugiproject/tsugi-util/pull/4

### CODE REVIEW

File: basiclti/tsugi-util/src/java/org/tsugi/pox/IMSPOXRequest.java [hunk 1]
```diff
@@ -173,7 +176,18 @@ public IMSPOXRequest(String bodyString)
...
+		try {
+			MimeType mimeType = new MimeType(contentType);
+			baseContentType = mimeType.getBaseType();
+		} catch (MimeTypeParseException e){
+			errorMessage = "Unable to parse mime type";
+			log.info("{}\n{}", errorMessage, contentType);
+			return;
+		}
+
+		if ( ! "application/xml".equals(baseContentType) ) {
```
Comment (mpellicer) [inline basiclti/tsugi-util/src/java/org/tsugi/pox/IMSPOXRequest.java]: I prefer using constants here instead of "application/xml", but looks good to me.

**Comment by baholladay** [comment]
Will this be merged into 19.x, 12.x and 11.x?

**Comment by ottenhoff** [comment]
Merges happen in JIRA once marked as Verified and Merge Status is set on the JIRAs

**Comment by baholladay** [comment]
Jira is set to verified and merge statuses are set

**Comment by baholladay** [comment]
is the PR not able to be merged into 11.x?

**Pull request merged**